docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -650,28 +650,50 @@ The following command operators are supported:
 
   UDP
     For :ref:`UDP Proxy <config_udp_listener_filters_udp_proxy>`,
-    NAMESPACE should be always set to "udp.proxy", optional KEYs are as follows:
+    if NAMESPACE is set to "udp.proxy.session", optional KEYs are as follows:
```

Comment:
`When NAMESPACE ...`

Reply:
Maybe `NAMESPACE should be always set to "udp.proxy.session"`

Reply:
> `When NAMESPACE ...`

OK, done, thanks!

Reply:
> when `NAMESPACE is set to "udp.proxy.session"`

when users want to log stats recorded in sessions. And it makes sure the tokens like `bytes_sent` can be reused.